### PR TITLE
feat: support Bot API 6.7

### PR DESCRIPTION
**Pull request by KnorpelSenf**
https://core.telegram.org/bots/api#april-21-2023

### CODE REVIEW

**Review by KnightNiwrem — APPROVED** [review]

**Comment by KnightNiwrem** [comment]
Will require re-approval after @grammyjs/types is updated, and import versions are also updated here.

File: src/convenience/keyboard.ts [hunk 1]
```diff
@@ -415,6 +417,27 @@ export class InlineKeyboard {
...
+     * Your bot will in turn receive updates for inline queries. You can listen
+     * to inline query updates like this:
+     * ```ts
+     * bot.on('inline_query', ctx => { ... })
+     * ```
+     *
+     * @param text The text to display
+     * @param query The query object describing which chats can be picked
+     */
+    switchInlineChosenChat(
```
Comment (KnightNiwrem) [inline src/convenience/keyboard.ts:435]: I'm not at all bothered by this but just noting..

I guess the naming convention `switch_inline_query` -> `switchInline` and `switch_inline_query_current_chat` -> `switchInlineCurrent` tends to omit `query` and `chat`. By that pattern, I suppose the conforming name would be more like `switchInlineChosen` instead. /shrug

File: src/platform.deno.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ if (isDeno) {
 
 // === Export system-specific operations
 // Turn an AsyncIterable<Uint8Array> into a stream
-export { readableStreamFromIterable as itrToStream } from "https://deno.land/std@0.178.0/streams/mod.ts";
+export { readableStreamFromIterable as itrToStream } from "https://deno.land/std@0.181.0/streams/mod.ts";
```
Comment (dcdunkan) [inline src/platform.deno.ts:20]:
```suggestion
export { readableStreamFromIterable as itrToStream } from "https://deno.land/std@0.184.0/streams/mod.ts";
```

File: test/deps.test.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 export {
     readAll,
     readerFromIterable,
-} from "https://deno.land/std@0.178.0/streams/mod.ts";
+} from "https://deno.land/std@0.181.0/streams/mod.ts";
```
Comment (dcdunkan) [inline test/deps.test.ts:4]:
```suggestion
} from "https://deno.land/std@0.184.0/streams/mod.ts";
```

File: test/deps.test.ts [hunk 4]
```diff
@@ -10,16 +10,16 @@ export {
     assertRejects,
     assertStringIncludes,
     assertThrows,
-} from "https://deno.land/std@0.179.0/testing/asserts.ts";
+} from "https://deno.land/std@0.181.0/testing/asserts.ts";
```
Comment (dcdunkan) [inline test/deps.test.ts:13]:
```suggestion
} from "https://deno.land/std@0.184.0/testing/asserts.ts";
```

File: test/deps.test.ts [hunk 5]
```diff
@@ -10,16 +10,16 @@ export {
...
 export {
     afterEach,
     beforeEach,
     describe,
     it,
-} from "https://deno.land/std@0.179.0/testing/bdd.ts";
+} from "https://deno.land/std@0.181.0/testing/bdd.ts";
```
Comment (dcdunkan) [inline test/deps.test.ts:19]:
```suggestion
} from "https://deno.land/std@0.184.0/testing/bdd.ts";
```

File: test/deps.test.ts [hunk 6]
```diff
@@ -10,16 +10,16 @@ export {
...
 export {
     type Spy,
     spy,
     type Stub,
     stub,
-} from "https://deno.land/std@0.179.0/testing/mock.ts";
+} from "https://deno.land/std@0.181.0/testing/mock.ts";
```
Comment (dcdunkan) [inline test/deps.test.ts:25]:
```suggestion
} from "https://deno.land/std@0.184.0/testing/mock.ts";
```

**Review by dcdunkan — APPROVED** [review]
LGTM. But its good to have the dependencies up-to-date.

File: src/convenience/keyboard.ts (hunk 1 above)
Comment (KnorpelSenf) [reply src/convenience/keyboard.ts:435]: Right, will rename. Thanks!

**Comment by KnorpelSenf** [comment]
Deps will be updated after the types release.

**Review by KnightNiwrem — APPROVED** [review]

**Pull request merged**